develop/reference_workarounds.rst

```diff
@@ -67,6 +67,65 @@ Although you can't join such tables directly, by wrapping the local table in a s
 
 Remember that the coordinator will send the results in the subquery or CTE to all workers which require it for processing. Thus it's best to either add the most specific filters and limits to the inner query as possible, or else aggregate the table. That reduces the network overhead which such a query can cause. More about this in :ref:`subquery_perf`.
 
+.. _upsert_into_select:
+
+INSERT…SELECT upserts
+---------------------
+
+Currently "on conflict" upserts are not supported on the coordinator for "insert into … select" statements. Attempting to upsert into distributed tables this way produces an error:
```

Comment:
We should probably expand this a bit. INSERT/SELECT has two modes: 1) It gets pushed down entirely if we are grouping by the partition column. or 2) We run the select on the workers, pull data to the coordinator, and then run inserts from the coordinator to the workers again. This is chosen if we aren't grouping by the partition column.

Its option 2 above which doesn't support ON CONFLICT. So we should clarify that and recommend modeling the data/choosing partition column so we can choose option 1. (Since that is also most efficient). And offer this workaround as a backup if data modeling isn't possible.